lib/le/host/connection.rb

```diff
@@ -198,24 +181,19 @@ def openConnection
           socket = TCPSocket.new(host, port)
 
           if @ssl
-	    cert_store = OpenSSL::X509::Store.new
-	    cert_store.set_default_paths
+            cert_store = OpenSSL::X509::Store.new
+            cert_store.set_default_paths
 
-            ssl_context = OpenSSL::SSL::SSLContext.new()
-	    ssl_context.cert_store = cert_store
+            ssl_context = OpenSSL::SSL::SSLContext.new
+            ssl_context.cert_store = cert_store
 
-            ssl_version_candidates = [:TLSv1_2, :TLSv1_1, :TLSv1]
-            ssl_version_candidates = ssl_version_candidates.select { |version| OpenSSL::SSL::SSLContext::METHODS.include? version }
-            if ssl_version_candidates.empty?
-                raise "Could not find suitable TLS version"
-            end
-	    # currently we only set the version when we have no choice
-            ssl_context.ssl_version = ssl_version_candidates[0] if ssl_version_candidates.length == 1
+            ssl_context.min_version = OpenSSL::SSL::TLS1_VERSION
```

Comment:
Should we still support TLS 1.0? I removed support for TLS 1.0 from our Python library at the beginning of the year, too.